build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -18,366 +18,50 @@
 ###########################################################################
 # rtl8721csm_download.sh
 
-## For manual port selection, set $3 as the portname, eg. make download ALL x ttyUSB2
-if [ -n "$3" ]; then
-	PORT="$3"
-else
-	PORT="ttyUSB1"
-fi
-
-WARNING="\n Port $PORT is selected\n\n
-	############################################\n
-	WARNINGS:\n
-	1. Make sure the board is in DOWNLOAD MODE.\n
-	2. Make sure NO other application like putty,\n
-	is occupying $PORT.\n
-	############################################\n"
+BOARD="RTL8721CSM"
+DEFAULT_PORT="ttyUSB1"
```

Comment:
I think it is better to support user specific port like before.
Sometimes, it is not connected to ttyUSB1.